### PR TITLE
Fix German translation No.2

### PR DESCRIPTION
**Pull request by erazortt**
Fix the German translation, Part 2 of 8
See also https://github.com/ihhub/fheroes2/issues/7860
And https://github.com/ihhub/fheroes2/pull/7865

### CODE REVIEW

File: files/lang/de.po [hunk 1]
```diff
@@ -589,7 +588,7 @@ msgid "Defense"
 msgstr "Verteid."
 
 msgid "Spell Power"
-msgstr "Z-Kraft"
+msgstr "Zauberkraft"
```
Comment (Laserlicht) [inline files/lang/de.po:591]: Can you verify if the text fits in the area? As far as I know this was extra abbreviated.

File: files/lang/de.po [hunk 2]
```diff
@@ -601,7 +600,7 @@ msgid "Luck"
 msgstr "Glück"
 
 msgid "Spell Points"
-msgstr "ZS-Punkte"
+msgstr "Zauberpunkte"
```
Comment (Laserlicht) [inline files/lang/de.po:603]: same here

File: files/lang/de.po [hunk 3]
```diff
@@ -1384,63 +1371,62 @@ msgid ""
...
 msgid ""
 "Destroy the barbarians who are attacking the southern border of your "
 "kingdom! Recover your fallen towns, and then invade the jungle kingdom. "
 "Leave no enemy standing."
 msgstr ""
-"Zerstören Sie die Barbaren, die die Südgrenze Ihres Königreichs angreifen! "
-"Holen Sie sich Ihre gefallenen Städte zurück und dringen Sie dann in das "
-"Dschungelkönigreich ein. Lassen Sie keinen Feind stehen."
+"Vernichtet  die Barbaren, die die Südgrenze Eures Königreichs angreifen! "
+"Holet sich Eure gefallenen Städte zurück und dringt dann in das "
```
Comment (Laserlicht) [inline files/lang/de.po:1407]:
```suggestion
"Holet Eure gefallenen Städte zurück und dringt dann in das "
```

File: files/lang/de.po [hunk 4]
```diff
@@ -1384,63 +1371,62 @@ msgid ""
...
 "so they will send you wood every 2 weeks. You must complete your mission "
 "before the first day of the seventh month, or the kingdom will surely fall."
 msgstr ""
-"Gewinnen Sie die Gunst der Elfen. Sie werden nicht zulassen, dass Bäume "
-"gefällt werden, also werden sie Ihnen alle 2 Wochen Holz schicken. Sie "
-"mussen Ihre Mission vor dem ersten Tag des siebten Monats abschließen, sonst "
-"wird das Königreich mit Sicherheit fallen."
+"Gewinnet die Gunst der Elfen. Sie werden nicht zulassen, dass Bäume gefällt "
+"werden, und werden Euch daher alle 2 Wochen Holz schicken. Beendet Eure "
+"Mission vor dem ersten Tag des siebten Monats, oder das Koenigreich wird "
```
Comment (Laserlicht) [inline files/lang/de.po:1421]:
```suggestion
"Mission vor dem ersten Tag des siebten Monats, oder das Königreich wird "
```

File: files/lang/de.po [hunk 5]
```diff
@@ -1384,63 +1371,62 @@ msgid ""
...
 
 msgid ""
 "This is the final battle against your rival kingdom of Harondale. Eliminate "
 "everyone, and don't lose the hero Jarkonas VI."
 msgstr ""
-"Dies ist die letzte Schlacht gegen Ihren rivalisierendes Königreich "
-"Harondale. Beseitigen Sie alle Fiende und verlieren Sie nicht Ihren Helden "
-"Jarkonas VI."
+"Dies ist die letzte Schlacht gegen das rivalisierendes Königreich Harondale. "
+"Beseitiget alle Fiende und verliert nicht Euren Helden Jarkonas VI."
```
Comment (Laserlicht) [inline files/lang/de.po:1429]:
```suggestion
"Beseitigt alle Feinde und verliert nicht Euren Helden Jarkonas VI."
```

**Review by Laserlicht — CHANGES_REQUESTED** [review]

File: files/lang/de.po (hunk 2 above)
Comment (erazortt) [reply files/lang/de.po:603]: I checked that as well: 
grafik

**Review by Laserlicht — APPROVED** [review]

**Review by Laserlicht — APPROVED** [review]

**Comment by erazortt** [comment]
@Laserlicht I think you need to approve the pull request, not only the changes. Or who can approve review?

**Comment by Laserlicht** [comment]
Yea, but I haven't the permission. :)

**Comment by ihhub** [comment]
@erazortt , a pull request can be approved an merged only by the core team (collaborators and owners). Please be patient as the team has limited time resources.

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@erazortt , a bunch of thanks for this update!

**Pull request merged**